test/dlc_tests/sanity/test_safety_check.py

```diff
@@ -82,8 +82,7 @@ def _get_safety_ignore_list(image_uri):
     additional_skips = []
     if is_canary_context():
         if (framework == "tensorflow" and "2.1" in image_uri) or \
-                (framework == "pytorch" and "1.4" in image_uri) or \
-                (framework == "pytorch" and job_type == "training" and "1.5" in image_uri):
+                (framework == "pytorch" and "1.4" in image_uri):
```

Comment:
nit: these changes won't actually do anything since the safety test is not running in canaries. I believe this cve is for awscli - you can remove the whole `if is_canary_context():` block

Reply:
Ah, alright. I was trying to find the actual CVE for the vulnerability here, as it wasn't documented in the code.

I'll remove it. Thanks.